### PR TITLE
chore: Move UniqueSelectionDispatcher to CDK.

### PR DESCRIPTION
**Pull request by josephperrott**
For #6913

### CODE REVIEW

File: src/lib/core/core.ts [hunk 1]
```diff
@@ -82,12 +82,6 @@ export {
   A11yModule,
 } from '@angular/cdk/a11y';
 
-export {
-  UniqueSelectionDispatcher,
-  UniqueSelectionDispatcherListener,
-  UNIQUE_SELECTION_DISPATCHER_PROVIDER,
-} from './coordination/unique-selection-dispatcher';
-
```
Comment (crisbeto) [inline src/lib/core/core.ts]: For backwards-compat, this should continue re-exporting the symbols from the CDK (see the `A11yModule` above).

Comment (josephperrott) [reply src/lib/core/core.ts]: Will do

**Comment by josephperrott** [comment]
@jelbourn When you have a chance to review this one, this is the other blocker to moving accordion to CDK.

**Comment by jelbourn** [comment]
@josephperrott add the merge-ready label when you're done with the cleanup

**Comment by kara** [comment]
@josephperrott Check out linter failure? https://travis-ci.org/angular/material2/jobs/281109389

**Comment by josephperrott** [comment]
@kara Should be good to go now.  Silly rebasing mistake by me...

**Pull request merged**